workflows/rules/variantcalling/ascat.smk

```diff
@@ -0,0 +1,107 @@
+# FILE: check_bam_existence.smk
+
+rule allele_count:
+    input:
+        bam = "{stype}/realign/{sname}_REALIGNED.bam"
+    singularity:
+        pipeconfig['rules']['ascat']['singularity']
+    params:
+        allele_count_script = f"{ROOT_DIR}/workflows/scripts/ascat/allele_count.R",
```

Comment:
Hm. Have a look at how we are using scripts at other instances in the pipeline or in other snakemake pipelines (e.g. twist-solid or poppy). Because it doesn't feel right to use ROOT_DIR in a snakemake pipeline? or i haven't seen it before at least... maybe i am wrong here though.

Reply:
I usually use workflow.basedir, which points to the location where the snake file is

Reply:
I initially wanted to use just "workflows/scripts/ascat/allele_count.R" (or add that to a config), but then saw [this commit](https://github.com/ClinicalGenomicsGBG/wgs_somatic/commit/e44bd06a9b97f803a300b1eaca7f36f5df206b4e) where many of the paths from the snakefile basedir were changed to f"{ROOT_DIR}/.....etc". I don't know why and if the usage of ROOT_DIR is better than just using the relative path.

Reply:
I guess the ROOT_DIR you can set in the definitions.py in case you want to change where snakemake finds the configs, workflows/rules, workflows/scripts